entries/jQuery.fx.interval.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0"?>
-<entry type="property" name="jQuery.fx.interval" return="Number">
+<entry type="property" name="jQuery.fx.interval" return="Number" deprecated="3.0">
```

Comment:
It's not deprecated in 3.0 but will be in 3.2. If there's a category missing, it should be added.

Do we need to look into things currently tagged as `deprecated="3.0"`? May there be more things to correct there?